src/PrestaShopBundle/Resources/views/Admin/Module/tab-modules-list.html.twig

```diff
@@ -23,6 +23,8 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 {% if modulesList is defined and modulesList is not empty %}
+<script src="{{ asset('themes/default/js/bundle/module/module_card.js') }}"></script>
```

Comment:
Indent

Reply:
Then I should fix the indent of the whole file ?
https://github.com/matks/PrestaShop/blob/767a185f443c6cbbc59dac2036f4f0961a7b286e/src/PrestaShopBundle/Resources/views/Admin/Module/tab-modules-list.html.twig